_studio/shared/umc/codec/vc1_dec/include/umc_vc1_dec_task_store.h

```diff
@@ -182,14 +182,11 @@ namespace UMC
         };
 
         // external memory management. No need to delete memory
-        void operator delete(void *p) THROWSEXCEPTION
+        void operator delete(void *)
         {
-            //Anyway its incorrect when we trying free null pointer
-            if (!p)
-                throw VC1Exceptions::vc1_exception(VC1Exceptions::mem_allocation_er);
         };
 
-        void operator delete(void *, void *) THROWSEXCEPTION
+        void operator delete(void *, void *) noexcept (false)
```

Comment:
We start to circle thru changes! Why this suddenly again?! In prev. comment you said that this operator is not actually used at all. I was expecting to see this line:
`void operator delete(void *, void *);`
I.e. declaration without implementation.

Reply:
@IlyaSergeyev : you actually need to write this:
`void operator delete(void *, void *) = delete;`
In this way linker will be happy. Sorry for misleading in prev. comment, I thought you will catch an idea.